ddtrace/llmobs/_utils.py

```diff
@@ -473,3 +476,136 @@ def on_openai_agent_span_finish(self) -> None:
         since output guardrails are only linked to the last LLM span for a particular agent.
         """
         self._last_llm_span = None
+
+
+class ExportSpansAPIError(Exception):
+    """Error raised when exporting spans from the LLMObs Export API."""
+
+    pass
+
+
+class LLMObsExportSpansClient:
+    """Retrieves spans from the LLMObs Export API."""
+
+    ENDPOINT = "/api/v2/llm-obs/v1/spans/events"
+    TIMEOUT = 10.0
+    DEFAULT_PAGE_LIMIT = 100
+
+    def __init__(
+        self,
+        api_key: str,
+        app_key: str,
+        site: str,
+    ) -> None:
+        self._api_key: str = api_key
```

Comment:
curious - can we reuse any of the BaseLLMObsWriter parent class setup/methods here?